core/modules/catalog/store/category/actions.ts

```diff
@@ -305,11 +309,20 @@ const actions: ActionTree<CategoryState, RootState> = {
     }
     return productPromise
   },
+  addAvailableFilter ({commit}, {key, options} = {}) {
```

Comment:
shouldn't be addFiler? can we add unavailable filters?

I'm not sure if it's responsible for adding filter to the list of available filters or using the filter to filter date. If 2nd option It should eb sth like `useFilter`

Reply:
@filrak in state we have something like this:
```
filters: {
      available: {},
      chosen: {}
    },
```
so it's in fact adding available filters, because you can also add chosen filters. I didn't want to change any logic, only to mutate state through actions and mutations :)